recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -27,3 +28,7 @@ do_install:append() {
 
     chmod 0600 ${D}${sysconfdir}/NetworkManager/system-connections/network?.nmconnection
 }
+
+do_install:append:ti-soc() {
```

Comment:
I think we should do this generally, not just for ti-soc

Reply:
Rationale: on BSP 7 this issue will happen for other SoMs that use the mwifiex driver

Reply:
Gotcha. Thanks for the feedback @dolcini, I'll make it generic.